### DISCUSSION TITLE
Notification Area Settings not saving

### DISCUSSION THREAD

**Discussion by Frrreddie1979** [valinet/ExplorerPatcher#3579] [Q&A]
In the settings for the notification area (Win 10 style), I can change the setting for the Slack icon from "only show notifications" to "show icon and notifications" but after saving, nothing happens and when entering the settings again, it's back to "only show notifications". Do you have any idea why this is not working?

**Comment by pyrates999** [comment]
You can drag and drop the icon in the system tray to the popup menu that you don't want shown.

duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1454

Please close this.

**Comment by Frrreddie1979** [comment]
Thanks for the hint with #1454. I'll close this one.
